launcher/ui/themes/ThemeManager.cpp

```diff
@@ -37,7 +37,8 @@
 ThemeManager::ThemeManager()
 {
     QIcon::setFallbackThemeName(QIcon::themeName());
-    QIcon::setFallbackSearchPaths(QIcon::themeSearchPaths());
+    QIcon::setThemeSearchPaths(QIcon::themeSearchPaths() << m_iconThemeFolder.path() << QStringLiteral(":/icons"));
```

Comment:
```suggestion
    QIcon::setThemeSearchPaths(QIcon::themeSearchPaths() << m_iconThemeFolder.path());
```
We might not need this, as the [docs](https://doc.qt.io/qt-6/qicon.html#themeSearchPaths) say

> All platforms will also have the resource directory :\icons as a fallback.

Reply:
fixed